mitodl/micromasters
# Stop automated updates of edX data for users in certain programs

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes #5124 

#### What's this PR do?
Stop automated updates of edX data for users in certain programs

#### How should this be manually tested?
Just need to confirm that only non DEDP( with no other enrollment but only in DEDP program) users are being passed to update edX data

You can change the flag to see if it's being sent to update edX for DEDP users if  `UPDATE_EDX_DATA_FOR_DEDP_PROGRAM_USERS=True`
